ci/docker/aarch64-unknown-linux-gnu/Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM rust-aarch64-unknown-linux-gnu
 
+RUN apt-get update && apt-get install -qy gcc-multilib libclang-dev \
```

Comment:
Maybe just add a comment here (also in the Dockerfiles) on why we need this stuff?

Reply:
@djc Done!